tests/test_events/test_slack_notifier.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+from unittest.mock import Mock, patch
+from helpers.event.event_slack_notifier import notify
+
+class test_slack_notifier(unittest.TestCase):
+
+    @patch('helpers.event.event_slack_notifier.requests.get')
+    def test_add_room(self, mock_get_request):
```

Comment:
Why is this function called `test_add_room`? You need to pay more attention to delay. Avoid unnecessary copying and pasting.